client/src/components/AuthHeader/useStyles.ts

```diff
@@ -30,6 +35,23 @@ const useStyles = makeStyles((theme) => ({
     boxShadow: 'none',
     marginRight: 35,
   },
+  tattooArt: {
+    fontSize: '2em',
+    marginLeft: 20,
+    letterSpacing: '.4em',
+  },
+  loginButton: {
+    color: 'white',
+    borderColor: 'white',
```

Comment:
You can use the CSS Border - Shorthand Property here :)

Reply:
There were 3 of these in total. They have all been fixed.